pubsub/gcloud/aio/pubsub/subscriber_message.py

```diff
@@ -46,5 +50,6 @@ def to_repr(self) -> Dict[str, Any]:
                 'attributes': self.attributes,
                 'data': base64.b64encode(self.data),
                 'publishTime': self.publish_time.strftime('%Y-%m-%dT%H:%M:%SZ')
-            }
+            },
+            'deliveryAttempt': self.delivery_attempt
```

Comment:
that... is not how master code looks like https://github.com/talkiq/gcloud-aio/blob/master/pubsub/gcloud/aio/pubsub/subscriber_message.py#L50
what happened here?

Reply:
Welp that's why there's merge conflicts. Lemme rebase

Reply:
Fixed thanks!